clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -3346,6 +3367,7 @@ raw2trace_t::write(raw2trace_thread_data_t *tdata, const trace_entry_t *start,
                     prev_was_encoding = false;
             }
             if (it->type == TRACE_TYPE_MARKER) {
+                // NOCHECK 2ndary processing
```

Comment:
Anything else to be done for this NOCHECK? I guess this was an alternate control point for fixing the timestamp, but process_marker works better?